Tutorial_1b_Moment_calculation.ipynb

```diff
@@ -0,0 +1,863 @@
+{
```

Comment:
<p>"that can we" —&gt; "that we can" (^-^)</p><p>"the emisison" —&gt; "the emission"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Smithsonian/sma-science-tutorials/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>